aliyun-ecs/src/main/java/org/jclouds/aliyun/ecs/features/VPCApi.java

```diff
@@ -121,19 +124,16 @@ public VPCs(Map<String, Iterable<VPC>> content, Integer pageNumber, Integer tota
    @Named("vpc:create")
    @POST
    @QueryParams(keys = "Action", values = "CreateVpc")
-   @Fallback(Fallbacks.NullOnNotFoundOr404.class)
    VPCRequest create(@QueryParam("RegionId") String region);
 
    @Named("vpc:create")
    @POST
    @QueryParams(keys = "Action", values = "CreateVpc")
-   @Fallback(Fallbacks.NullOnNotFoundOr404.class)
    VPCRequest create(@QueryParam("RegionId") String region, CreateVPCOptions vpcOptions);
 
    @Named("vpc:delete")
    @POST
    @QueryParams(keys = "Action", values = "DeleteVpc")
-   @Fallback(Fallbacks.NullOnNotFoundOr404.class)
```

Comment:
Actually, we use fallbacks on DELETE operations. If we want to delete a resource that is no longer there, just don't fail as we are already in the desired status.